Modules/HTTPServer/Sources/HTTPServer/Server/Server.swift

```diff
@@ -157,36 +157,46 @@ extension Server {
     }
 
     public func process(stream: Stream) throws {
-        let parser = RequestParser(stream: stream, bufferSize: bufferSize)
+        let chunkBytes = UnsafeMutablePointer<UInt8>.allocate(capacity: 4096)
+        defer { chunkBytes.deallocate(capacity: 4096) }
+        let chunk = UnsafeMutableBufferPointer<UInt8>(start: chunkBytes, count: 4096)
+        
+        let parser = Parser(mode: .request)
         let serializer = ResponseSerializer(stream: stream, bufferSize: bufferSize)
```

Comment:
Ooo I'll reuse that, we should probably default to 4kb or something though 2kb is kinda low?